beastling/clocks/baseclock.py

```diff
@@ -1,24 +1,24 @@
 from beastling.util import xml
+from beastling.util.misc import FromOptions
 
 
-class BaseClock(object):
+class BaseClock(FromOptions):
```

Comment:
Should this actually be an ABC?

Reply:
What's your definition of "ABC"? I don't see anything in here that would strictly require this class **not** to be used other than by inheriting.

Reply:
BaseClock doesn't have a branch rate model, which to me makes that method pretty abstract, so the whole class is? It also lacks __type__ and __distribution__, and declaring it abstract would prevent an accidental instantiation if after some refactoring `[clock] type=base` would theoretically run etc.